initfiles/sbin/hpcc_setenv.in

```diff
@@ -157,8 +157,8 @@ if [[ $platform == 'linux' ]]; then
             fi
     done
 
-    if [[ -d "/opt/rh/devtoolset-2" ]]; then
-        export CL_PATH=/opt/rh/devtoolset-2/root/usr
+    if [[ -d "/opt/rh/devtoolset-7" ]]; then
```

Comment:
This will make sure we use devtoolset7 when hpcc is compiling generated c++ - but I don't think it makes sure we use devtoolset7 when building the platform?

Reply:
Yes, this is the stuff that will be necessary at installation, runtime, and documentation.  Having an issue though with centos 6 that I ran into last evening when using the higher compiler, need to work around it before this can go live.  Libmemcached is giving me problems again and just building it from source using the higher compiler runs into compilation errors.